sequencer/src/bytes.rs

```diff
@@ -0,0 +1,193 @@
+//! Convenient serialization for binary blobs.
```

Comment:
This would make a nice standalone crate.

Reply:
Sure, I can split it out. Any suggestions for a name?

Reply:
You mean something that isn't taken on crates.io or just in general? Uhm... base64bytes?